src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        writeResultFile(toFileName, readAndCalculatedFile(fromFileName));
+    }
+
+    private String readAndCalculatedFile(String fileName) {
+        int[] results = new int[3];
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] array = line.split(",");
+                if (array.length == 2) {
```

Comment:
Instead of using 'split()', consider defining a constant for the comma delimiter to improve code readability.